### ISSUE TITLE
Laucher ignoring SIGINT

### ISSUE THREAD

**Issue by victordejong** [kpcyrd/spotify-launcher#63]
### Description

When running spotify-launcher from the command line, the launcher wrapper and spotify itself ignore SIGINT completely. This leaves you with no option to abort when spotify and/or the launcher are misbehaving.

### Expected behaviour

When a SIGINT is issued, the launcher closes spotify and exits.

### Impact

Minimal, command line is only used during development.

### Steps to reproduce

1. Run spotify-launcher from the command line:
```bash
spotify-launcher
```
2. Wait until spotify has settled, then issue a SIGINT (Ctrl+C by default). Observe that spotify does not exit and the laucher hangs.

### Affected versions
 * v0.6.0
 * v0.6.1

**Comment by victordejong** [comment]
Issue confirmed present in v0.6.1

**Comment by kpcyrd** [comment]
I think there's a problem elsewhere, spotify-launcher doesn't register any signal handlers, so SIGINT is just as fatal as SIGKILL/-9. I tested this and the launcher terminates immediately on ctrl-c.

What the spotify binary itself does is intentionally left up to spotify.

**Comment by victordejong** [comment]
You're correct, I'm fairly sure this is due to the signal handling of Spotify itself and not a fault of the spotify-laucher signal handling.

I just thought it might by good UX to add this functionality to the launcher as Spotify itself seems to ignore the SIGINT signal, thus "hanging" when ran from the CLI.
